src/gui/app.cpp

```diff
@@ -97,6 +97,10 @@ void App::showUi() noexcept
 {
 	ShellOptions opts{ GetShellOptionsFromQSettings() };
 
+	if (m_parser.isSet("name")) {
+		setApplicationName(m_parser.value("name"));
```

Comment:
Wouldn't this cause QSettings to also be affected? This would mean config files would use the wrong paths.

Reply:
I think it would only affect the default CTOR. Since we call `QSettings{ "nvim-qt", "nvim-qt")` explicitly it should not have any impact.